sdk/options.go

```diff
@@ -183,3 +185,11 @@ func WithNoKIDInKAO() Option {
 		c.tdfFeatures.noKID = true
 	}
 }
+
+// WithKIDInNano disables storing the KID in the KAS ResourceLocator.
+// This allows generating NanoTDF files that are compatible with legacy file formats (no KID).
+func WithKIDInNano() Option {
+	return func(c *config) {
+		c.tdfFeatures.nanoKID = false
+	}
+}
```

Comment:
@pflynn-virtru Shouldn't this be a nanotdf config option?

Reply:
oh, something like this?
```
	tdfFeatures             tdfFeatures
	nanoFeatures         nanoFeatures
```

Reply:
done